readme.md

```diff
@@ -34,65 +34,69 @@ Once installation is working, take a look at the existing code to make sure you
 
 ### Instructions & Guiding Questions
 
-- [ ] Take a look at the `db/seeds.js` file.
+- [x] Take a look at the `db/seeds.js` file.
 
 * **Question:** Describe what this code is doing and what its purpose is.
 
-* **Your Answer:** 
-
+* **Your Answer:**
+Clear the db for dev purposes! How nice.
 ---
 
-- [ ] Imagine that as a user, you enter your username and password into a site in order to signup.
+- [x] Imagine that as a user, you enter your username and password into a site in order to signup.
 
 * **Question:** What happens next?
 
 * **Your Answer:**
-
+The information entered into the auth form needs to be sent to the server, the username needs to be validated as unique (send back an error if not), if there is validation necessary for the pw it needs to be checked that it is valid (send back an error if not). Then if those conditions are met, the user and related info needs to be stored in some kind of database (maybe encrypted??) where it can be validated against in the future when the user signs in.
 ---
 
-- [ ] Imagine that as a user, you are now logging back into that same website. 
+- [x] Imagine that as a user, you are now logging back into that same website.
 
 * **Question:** How does the website verify that you are indeed the same user?
 
 * **Your Answer:**
-
+Compare username and password?
 ---
 
-- [ ] Imagine that as a logged-in user, you try to go to a route you are not supposed to (e.g. /admin).
+- [x] Imagine that as a logged-in user, you try to go to a route you are not supposed to (e.g. /admin).
 
 * **Question:** How does the website know you are or are not allowed on a specific route?
 
 * **Your Answer:**
+Usually there would be specific roles/rights assigned to a given user (maybe a default assigned when you sign up). The site would then check against those roles/rights and throw and 403 if you don't have permission.
 
 * **Question:** Describe the difference between authentication and authorization.
 
 * **Your Answer:**
-
+authentication -
+authorization -
 ---
 
-- [ ] Build a new model called `Guest`. The `Guest` model should have the following fields: `username`, `password`
+- [x] Build a new model called `Guest`. The `Guest` model should have the following fields: `username`, `password`
 
 ---
 
-- [ ] Create a new route at `POST /api/signup` at `/api/routes/auth.js`. This route should:
+- [x] Create a new route at `POST /api/signup` at `/api/routes/auth.js`. This route should:
   1. Create a new `Guest`, pulling `username` and `password` from the request body
   1. Return a success message with the user's information (for now)
 
 * **Question:** This code is currently _very_ insecure. Why?
 
 * **Your Answer:**
+Just saving a plaintext password!
 
 * **Question:** What would happen if three different users tried to sign up with the same username? How can we prevent that?
 
 * **Your Answer:**
+Validate a match in the db before committing a new document
```

Comment:
Currently all three could be created with Identical usernames! Adding a unique validation before create is necessary.